Community_Guidelines.md

```diff
@@ -0,0 +1,380 @@
+- Open source project launches in #github
+- Job postings in #jobs-board (discussion in #jobs-chat)
+
+**Not Allowed:**
+- Repeated self-promotion
+- Off-topic promotion
+- Promotional content disguised as help
+
+### Screenshots and Recordings
+
+**Privacy Considerations:**
+- Blur or redact sensitive information (names, emails, IPs, etc.)
+- Don't share screenshots of DMs without permission
+- Don't share internal company information
+- Assume workspace conversations are public
+
+**Recording Policy:**
+- Don't record calls without consent
+- Don't share recordings of Slack calls externally
+- Admin team may record calls for training purposes (with notice)
+
+## Accessibility and Inclusion
+
+### Making Content Accessible
+
+- Add alt text to images when possible
+- Use descriptive link text, not "click here"
+- Follow channel threading norms
+- Avoid wall-of-text messages
+- Use headings in longer posts
+
+### Language and Communication
+
+- Use clear, simple language
+- Define acronyms on first use
+- Be patient with non-native English speakers
+- Avoid sarcasm - it doesn't translate well
+- Use emoji thoughtfully (some are culturally specific)
+
+### Neurodiversity
+
+- Some members may communicate differently
+- Be patient with communication styles
+- Assume good intent
+- Offer clarification without judgment
+
+## Topics to Approach Carefully
+
+### Generally Acceptable (But Be Respectful)
+
+- Apple platform technical discussions (primary purpose)
+- IT and systems administration
+- Career advice and development
+- Conference and meetup announcements
+- Industry news relevant to Mac admins
+
+### Requires Caution
+
+**Politics:**
+- There is a private #politics channel available
+- Request access by DMing an Admin (see placeholder channel #politics)
+- General channels: keep politics relevant to our work (tech policy, privacy laws)
+- Admins may intervene if discussions become disruptive
+
+**Religion:**
+- Respect all beliefs and non-beliefs
+- Keep discussions civil
+- Don't proselytize
+- Admins may intervene if discussions become contentious
+
+**Controversial Topics:**
+- Use content warnings for potentially triggering topics
+- Consider whether Slack is the right venue
+- Move to DMs if requested
+- Stop if asked by Admins
+
+### Not Appropriate for This Workspace
+
+- Cryptocurrency promotion or investment advice
+- Multi-level marketing (MLM)
+- Political activism unrelated to tech
+- Personal grievances with employers (use discretion)
+- Legal advice (we're not lawyers)
+- Medical advice (we're not doctors)
+
+## Bots and Automation
+
+Only MAF-approved bots are enabled in the workspace. User bots and custom integrations are not permitted.
+
+### Slash Commands and Integrations
+
+- Use built-in Slack commands appropriately
+- Report malfunctioning bots to Admins
+
+## Data and Privacy
+
+### Your Data
+
+- The Mac Admins Slack is on a Business+ plan with full message and file retention
```

Comment:
This is critical information that appears only in the Community Guidelines. This should be prominent in the main Code of Conduct or have its own privacy policy section. Members need to know upfront that nothing gets deleted.